README.md

```diff
@@ -1,139 +1,57 @@
 # Reliability Engineering IP Whitelisting Route Service
```

Comment:
```suggestion
# Reliability Engineering IP Safelisting Route Service
```

Reply:
Renamed to `GOV.UK PaaS IP authentication route service`